lib/validators.js

```diff
@@ -19,7 +19,7 @@ exports.clientSecret = Joi.string()
   .required();
 
 exports.codeVerifier = Joi.string()
-  .length(32); // https://tools.ietf.org/html/rfc7636#section-4.1
+  .regex(/^[A-Za-z0-9-_]{43,128}$/);  // https://tools.ietf.org/html/rfc7636#section-4.1
```

Comment:
Two tiny nits here:

* Code clarity might be improved by making this regex a shared constant in `validators.B64URL_STRING`
* We might get better error messages if we use `string().min(43).max(128).regex(validators.B64URL_STRING)`

But neither are blockers for me.

Reply:
I'm happy to make those changesmy tomorrow morning (US time).

Reply:
Yeah this is a pretty good suggestion, I will update it !